soc/riscv/openisa_rv32m1/Kconfig.defconfig

```diff
@@ -162,6 +162,14 @@ config I2C_RV32M1_LPI2C
 
 endif # I2C
 
+if SPI
+
+config SPI_RV32M1_LPSPI
+	def_bool y
```

Comment:
s/def_bool/default/

Reply:
Done, and thanks for looking out for things like that!